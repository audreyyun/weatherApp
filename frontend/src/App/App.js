import React from 'react';
import WeatherContainer from '../components/Weather/WeatherContainer';
// import '../scss/style.scss'

const App = () => { 
  return(
    <div>
        <WeatherContainer/>
    </div>
  )
}

export default App;