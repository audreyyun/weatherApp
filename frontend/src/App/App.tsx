import React from 'react';
import Weather from '../components/Weather/Weather';
// import WeatherContainer from '../components/Weather/WeatherContainer';
// import '../scss/style.scss'


const App = () => { 
  return(
    <div>
        <Weather/>
        {/* <WeatherContainer/> */}
    </div>
  )
}

export default App;