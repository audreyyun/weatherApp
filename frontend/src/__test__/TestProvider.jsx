import React from 'react'
import { Provider } from 'react-redux';
import configureStore from '../store/store';

const store = configureStore();

const TestProvider = ({ children }) => (
    <Provider store={store}>
        {children}
    </Provider>
);

export default TestProvider;