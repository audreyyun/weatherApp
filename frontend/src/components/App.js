import React from 'react';
import { Route } from 'react-router-dom';
import WeatherContainer from './weather/weatherContainer';
import '../scss/style.scss'

const App = () => { 
  return(
    <div>
        <Route exact path="/" component ={WeatherContainer} />
    </div>
  )
}

export default App;