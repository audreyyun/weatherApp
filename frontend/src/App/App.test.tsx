import '@testing-library/jest-dom/extend-expect';
import React from 'react'
import App from './App'
import { render } from '@testing-library/react'
import TestProvider from '../__test__/testProvider';

const AppTest = () => (
  <TestProvider >
    <App />
  </TestProvider>
);


describe('App', () => {
  it('should render without crashing', () => {
    const component = render(<AppTest />)
    expect(component.asFragment()).toMatchSnapshot()
  })
})
