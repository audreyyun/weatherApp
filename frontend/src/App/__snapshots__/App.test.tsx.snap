// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
<DocumentFragment>
  <div>
    <div
      class="weatherPageContainer"
    >
      <h1
        class="appTitle"
      >
        Audrey's Weather App
      </h1>
      <div>
        <div
          class="locationForm"
        >
          <div
            class="locationInput"
          >
            <input
              class="zipcodeSearchInput input"
              placeholder=" Enter City Name"
            />
            <button
              class="locationInputBtn input"
            >
              See Weather
            </button>
          </div>
        </div>
        <div
          class="no-weather-info"
        >
          Please enter a city to see today's weather!
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
